core/src/main/scala/org/ensime/core/Completion.scala

```diff
@@ -376,9 +376,12 @@ object CompletionUtil {
     matchEntire: Boolean, caseSens: Boolean): Boolean = {
     val prefixUpper = prefix.toUpperCase
 
+    val prefixRegexp = "[A-Za-z0-9]*" + prefix.replaceAll("(?<!^)([A-Z])", "[A-Za-z0-9]*$1") + "[A-Za-z0-9]*"
+
     (matchEntire && m == prefix) ||
```

Comment:
nice. I keep meaning to allow the exclusion of a bunch of noise in here. using a custom `sysdef` (typelevel scala) helps a **lot**.